web-animations/interfaces/AnimationTimeline/document-timeline.html

```diff
@@ -20,6 +21,18 @@
 }, 'document.timeline identity tests');
 
 promise_test(function(t) {
+  var baseTime = document.timeline.currentTime;
+  var iframe = document.getElementById('iframe');
+  var iframeBaseTime = iframe.contentDocument.timeline.currentTime;
+  return waitForAnimationFrames(4).then(function() {
+    var delta = document.timeline.currentTime - baseTime;
+    var iframeDelta = iframe.contentDocument.timeline.currentTime - iframeBaseTime;
+    assert_greater_than(delta, 0);
+    assert_times_equal(delta, iframeDelta, 'different document timelines should progress at the same rate');
+  });
+}, 'document.timeline should tick consistently across frames');
```

Comment:
Ok, great. @alancutter do you mind giving me push access to this PR? I'm happy to rebase it, drop this first test, then tweak the last test to only wait 50ms instead of 100ms.